airframe-http/src/main/scala/wvlet/airframe/http/router/RedirectToRxEndpoint.scala

```diff
@@ -22,8 +22,31 @@ import wvlet.log.LogSupport
   * @param endpoint
   */
 class RedirectToRxEndpoint(endpoint: RxHttpEndpoint) extends LogSupport {
```

Comment:
@xerial Any potential side-effects? Any probability something would be relying on the method name or the fact that it only has a GET endpoint?